templates/invoices/new.html

```diff
@@ -375,6 +376,10 @@ <h2>Fakturadelar</h2>
                     this.expense_parts.splice(i, 1)
             }
         }
-    })
+    });
+
+    function uploadFile (filename) {
+        document.getElementById('filename').innerText = filename.split('\\').pop();
+    }
```

Comment:
This works for one file, but the input is `multiple`: it allows uploading several files, so maybe you want to show all of them. I suggest:

```suggestion
    function updateUploaded() {
    	const files = [...document.getElementById('file').files];
        document.getElementById('filename').innerText = files.map(f => f.name).join(', ');
    }
```

(plus `onchange="updateUploaded()"` above)

Reply:
Solved